fast64_internal/oot/f3d/properties.py

```diff
@@ -149,12 +150,12 @@ def draw_props(self, layout: UILayout, mat: Object, drawLayer: str):
 
 class OOTDefaultRenderModesProperty(PropertyGroup):
     expandTab: BoolProperty()
-    opaqueCycle1: StringProperty(default="G_RM_AA_ZB_OPA_SURF")
-    opaqueCycle2: StringProperty(default="G_RM_AA_ZB_OPA_SURF2")
-    transparentCycle1: StringProperty(default="G_RM_AA_ZB_XLU_SURF")
-    transparentCycle2: StringProperty(default="G_RM_AA_ZB_XLU_SURF2")
-    overlayCycle1: StringProperty(default="G_RM_AA_ZB_OPA_SURF")
-    overlayCycle2: StringProperty(default="G_RM_AA_ZB_OPA_SURF2")
+    opaqueCycle1: StringProperty(default="G_RM_AA_ZB_OPA_SURF", update=update_all_material_nodes)
```

Comment:
for readability can you add an indirection, like on props put `update=on_world_defaults_change`
and create the function `on_world_defaults_change` in which `update_all_material_nodes` is called

Reply:
If I'm gonna add something seperate I might as well only call update_rendermode_preset in there